src/bigfile.c

```diff
@@ -406,7 +404,8 @@ _big_block_open(BigBlock * bb, const char * basename)
     } else {
         /* The meta block of a big file, has on extra files. */
         bb->Nfile = 0;
-        strcpy(bb->dtype, "####");
+        strncpy(bb->dtype, "####", 5);
```

Comment:
Since we use a string literal as the source, I don't really see why line 408 is needed. If it is comig from clang checkers, then probably change this to memcpy(...., 5)?

Reply:
That is true, I will take it out! It is not clang checkers - I have seen in some frontera stack traces that the intel compiler is doing crazy things and I want to stop it.